web/src/lib/panel.svelte

```diff
@@ -1,7 +1,7 @@
 <script lang="ts">
 	import type { Optional } from "$lib/optional";
 
-	export let kind: "regular" | "emphasis" = "regular";
+	export let kind: "regular" | "emphasis" | "primary" = "regular";
```

Comment:
I don't fully understand the motivation for adding this when we already have `emphasis`?

Reply:
Because the way `emphasis` is currently used is with a call to action. The other name I had for this was `success`, which might actually make more sense, but I used `primary` since it's the same(-ish, I think I tweaked it slightly actually) colours as primary buttons.

Also it looks weird to me if there are two emphasis panels next to each other, like there would be in unpublished referendum results.